share/completions/zpaq.fish

```diff
@@ -0,0 +1,48 @@
+#zpaq 7.15
+
+#shortcut
+function _seen
+    __fish_seen_subcommand_from $argv
```

Comment:
For such an abbreviation, you could use local variable for the same effect. The variable does not linger in the global scope, unlike a function.
```fish
set -l seen __fish_seen_subcommand_from
complete -c zpaq -f -n "not $seen $commands" -a "$commands"
```